frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css

```diff
@@ -0,0 +1,60 @@
+.ArgumentsGrid {
+  margin: 1rem 0;
+  display: grid;
```

Comment:
:x: Color of numbers has changed
![image](https://github.com/user-attachments/assets/377521b9-91d7-4706-8c76-003faca81261)

Reply:
<img width="545" alt="image" src="https://github.com/user-attachments/assets/ae8b5527-6c24-47d8-afc6-81558e8f7316">

6eb9d18